rerun_py/rerun_sdk/rerun/blueprint/api.py

```diff
@@ -88,9 +89,9 @@ def __init__(
         self.origin = origin
         self.contents = contents
         self.visible = visible
-        self.properties = properties
-        self.defaults = defaults
-        self.overrides = overrides
+        self.properties = properties if properties is not None else {}
+        self.defaults = defaults if defaults is not None else []
+        self.overrides = overrides if overrides is not None else {}
```

Comment:
Sometimes I wish I was writing Lua and could just write `overrides or {}` instead of all this verbosity…

Reply:
Well you can, but I don't like it much because of how many things are falsy in python.